src/ts/app/domain/src/model.ts

```diff
@@ -0,0 +1,11 @@
+import {model} from "app/domain"
+
+export interface PythonModule {
+    name: string
+    content: string
+}
+
+export interface DocumentCollection {
```

Comment:
can we use the langugage from the already reviewed design doc? I think this is a CodeContainer (unless it's something else, then let me know)

https://docs.google.com/document/d/1JjA2RUAmmTnvOJGWkbq4ws1WsIVMFr2UI_hN6d9NS98/edit

Reply:
I'm definitely thinking in terms of that doc, and intend there to be a `has-a` relationship here - a `CodeContainer` `has-one` `DocumentCollection`.

I don't intend any of this to be some final expression of the domain model - everything represented here just helped me order my thoughts as I was working through the task (and during which I try to keep our design doc in mind, which I'm still 100% committed to). 

I would like to see the domain model sort of organically grow as we add PR's. Or rather, I just don't know how to develop a perfect model up front. To some extent I think a key test of our application code design is how productively we can quickly change/evolve the model as we realize there are better ways to represent various aspects.

It is really appropriate for the `editor` to be aware of multiple `Document`'s? Or should `editor` be focused on just the current `Document` and some other concept, like `editor-frame` (naming??) "knows" about the `DocumentCollection`? Or is that granularity-overkill? Or should `editor` "know" even more, like about the full `CodeContainer`? I admit I can't see forward that far.

Another good VC topic.